test/unit/testreporter.js

```diff
@@ -48,21 +48,37 @@ var TestReporter = function (browser) {
 
   this.jasmineStarted = function (suiteInfo) {
     this.runnerStartTime = this.now();
-    sendInfo("Started tests for " + browser + ".");
+
+    const total = suiteInfo.totalSpecsDefined;
+    const seed = suiteInfo.order.seed;
+    sendInfo(`Started ${total} tests for ${browser} with seed ${seed}.`);
```

Comment:
This is *really* nice, and one of those things that you can't help but wonder why we haven't done years ago!